### ISSUE TITLE
Extra_Transaction_Fee Error 

### ISSUE THREAD

**Issue by riderofthestorm77** [EvernodeXRPL/evernode-test-resources#7]
When putting in a fee for the "extra transaction fee" I encountered and error (see attachment).

I see it asks for value in XAH drops, but I dont think its intuitive since the previous step accepts 0.0001 price for the lease. When this step comes straight after, and apparently dont accept 0.05 even though thats greater or equal zero.

This might be a recurring discord theme :)

Beside this I had no issue during the installation.
<img width="591" alt="extra_transaction_fee" src="https://github.com/EvernodeXRPL/evernode-test-resources/assets/158591004/1ade0c2a-108b-44dd-9497-9094d9331f41">

**Comment by go140point6** [comment]
I agree, if in drops the installer should have an example of what that means, the average person isn't going to know really what drops refer to.  Maybe show what the current fee in drops is and suggest a doubling of that number as an example.

**Comment by gadget78** [comment]
i agree, showing what the current rate is, would be a good guide of what to put here ..
